ogr/ogrsf_frmts/sqlite/ogrsqlitedatasource.cpp

```diff
@@ -307,6 +308,138 @@ int OGRSQLiteDataSource::GetSpatialiteVersionNumber()
     return v;
 }
 
+/************************************************************************/
+/*                        LoadRelationships()                        */
+/************************************************************************/
+
+void OGRSQLiteDataSource::LoadRelationships() const
+
+{
+    m_osMapRelationships.clear();
+
+    if( hDB )
+    {
+        auto oResult = SQLQuery( hDB,
```

Comment:
bisecting a bit, pragma_foreign_key_list() is only available since sqlite 3.16.0 (which is admitedly rather old, 2017-01-02)